src/analytics/formTracker/formTracker.ts

```diff
@@ -303,14 +297,38 @@ export class FormTracker extends BaseTracker {
    * @return {FormField[]} array containing the combined date field
    */
   combineDateFields(fields: FormField[]): FormField[] {
-    const combineDateField: FormField[] = [
-      {
-        id: fields[0].id,
-        name: fields[0].id.split("-")[0],
-        value: `${fields[0].value}-${fields[1].value}-${fields[2].value}`,
-        type: "date",
-      },
-    ];
-    return combineDateField;
+    let newArrayFields: FormField[] = [];
+    fields.forEach((field) => {
+      //if dash is in the name
+      if (field.name.includes("-")) {
```

Comment:
This might be unnecessary, but wondering here if we should check if it includes `-day`, `-month` or `-year` if there is any possibility that there could be a `-` in a non-date field?

Reply:
We are calling this function only when we detected the field is a date field (because we have a day date fields, month date fields and year date fields), so the check is already done at this point.